pkg/cidata/cidata.TEMPLATE.d/boot/40-install-containerd.sh

```diff
@@ -5,6 +5,24 @@ if [ "${LIMA_CIDATA_CONTAINERD_SYSTEM}" != 1 ] && [ "${LIMA_CIDATA_CONTAINERD_US
 	exit 0
 fi
 
+if [ "${LIMA_CIDATA_CONTAINERD_SYSTEM}" = 1 ] && [ -f /etc/alpine-release ]; then
+	# need to use the containerd packages from alpine, because the
+	# installation from nerdctl requires glibc (alpine uses musl)
+	apk add runc containerd cni-plugins
```

Comment:
Should this be done in `30-install-packages.sh` and protected by a check that `containerd` or the plugins aren't already installed?

Reply:
According to my experiments, we need one set of plugins for containerd and one set of plugins for nerdctl

Reply:
Actually I think it is more or less the _same_ CNI plugins, but installed in different paths (/usr vs /usr/local)